build/ProjectReunion-BuildFoundation.yml

```diff
@@ -6,17 +6,17 @@ name: $(version)
 trigger: none
 
 variables:
-- template: WindowsAppSDK-Versions.yml
-- template: WindowsAppSDK-CommonVariables.yml
 - template: WindowsAppSDK-Foundation-TestConfig.yml@WindowsAppSDKConfig
+- template: AzurePipelinesTemplates\WindowsAppSDK-Versions.yml@WindowsAppSDKVersionConfig
```

Comment:
For my understanding, aside from the conceptual separation of "test" and "version" stuffs for clarity, are there other reasons for declaring a separate new "config repo" versus using the existing "WindowsAppSDKConfig" declaration?

Reply:
Thanks!